### ISSUE TITLE
Methods called by Call Node cannot create nodes.

### ISSUE THREAD

**Issue by Aerce** [coppolaemilio/dialogic#551]
## The problem

**Describe the bug**
Creating a node from a method called by Call Node does not work if there is no Wait or Text earlier in the timeline.

**To Reproduce**
1.Create a timeline with a Call Node event.
2. Call a function that creates a node.
Mine was a function placed in the parent of DialogNode:
```
func labelevents() :
	var e1_label = Label.new()
	e1_label.text = "test"
	e1_label.rect_position = Vector2(100,100)
	add_child(e1_label)
	print("Node path:" + (str(e1_label.get_path())))
	print("This part works regardless of placement")
```
3. The label does not appear when the game is run, but the print functions run fine.

**Expected behavior**
Call Node event functioning the same regardless of its position in the timeline.

**Screenshots**
![notworking](https://user-images.githubusercontent.com/91739232/135646358-fff02b36-f225-4daa-8d86-fd76b73851e9.jpg)
![working](https://user-images.githubusercontent.com/91739232/135646378-72865e46-8423-48c1-9820-859d822a45b7.jpg)

**System**
 - OS: Windows
 - Godot Version: 3.3.2, 3.3.4
 - Dialogic Version: 1.2.5, tried running main branch but the addon didn't work

## Solutions

**Workaround**

Adding a Wait or Text event before the Call Node event makes it work as intended. Character Join and Choice had no effect. Removing all Text and Wait events had no effect.

**Comment by GimmiRuski** [comment]
I was unable to replicate the issue on my end.

I set up a project verbatim to the one described and couldn't have the label show up. Following the path parameter revealed that the target node ends up being the canvas layer, so I'm not sure how you were able to add the label in the first place.

To achieve the expected behavior, I changed the target node from `..` to `../..`, which worked in both the orders shown in the screenshots.

One thing that occurred while trying to reproduce this issue was that the input in the node target field didn't correspond to what was passed as a parameter, but that could be caused by me, since I was editing dialogic's code while having it already loaded.
